core-bundle/composer.json

```diff
@@ -70,6 +70,7 @@
         "scheb/2fa-bundle": "^5.0",
         "scheb/2fa-trusted-device": "^5.0",
         "scssphp/scssphp": "^1.0",
+        "sensiolabs/ansi-to-html": "^1.1",
```

Comment:
This is a requirement of the installation bundle, not the core bundle, isn't it?

Reply:
Yes, my bad.